src/sass/_theme_rst.sass

```diff
@@ -223,9 +223,10 @@
   // Sphinx can highlight searched text with ?highlighted=searchterm
   .highlighted
     background: $highlight-color
-    display: inline-block
+    box-shadow: 0 0 0 3px $highlight-color
+    display: inline
     font-weight: bold
-    padding: 0 $base-line-height / 4
+    padding: initial
```

Comment:
Is this needed? I see no difference if I remove the line altogether, but there may be cases I did not check.